Database/Terraform/main.tf

```diff
@@ -0,0 +1,36 @@
+# This file terraforms the database on AWS RDS
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.AWS_REGION
+}
+
+# terraform import aws_db_instance.database c11-hermes-db
+resource "aws_db_instance" "database" {
+    allocated_storage                     = 20
+    db_name                               = "sales"
```

Comment:
I think a clearer database name would be nice

Reply:
I think also putting this into `tfvars` would help

Reply:
name changed and done